user_guide_src/source/libraries/uri.rst

```diff
@@ -138,31 +139,51 @@ can be used to manipulate it:
 Query
 -----
 
-The query variables can be manipulated through the class using simple string representations. Query values can only
+The query variables can be manipulated through the class using simple string representations.
```

Comment:
I think "query data" can be used instead of "query variable"

Reply:
Better! Done.